websocketpp/common/asio_ssl.hpp

```diff
@@ -33,7 +33,7 @@
 #ifdef ASIO_STANDALONE
     #include <asio/ssl.hpp>
 #else
-    #include <boost/asio/ssl.hpp>
```

Comment:
This side of the ifdef is the boost side and should not be altered.

Reply:
@paulbombarde @dpursey thank you for reviewing my code. 
I've closed this pull-request since apparently I can achieve the same behavior by just adding two defines (even outside websocketpp):

_WEBSOCKETPP_CPP11_SYSTEM_ERROR_
-D_ASIO_STANDALONE_